ocs_sample_library_preview/Security/AccessRights.py

```diff
@@ -0,0 +1,45 @@
+import json
```

Comment:
Do we need this class definition? I wouldn't think we would need it if it's literally just a `list[CommonAccessRightsEnum]`. We haven't built class definitions for other arrays like `list[SdsStream]`.

Reply:
I was on the fence about creating this class, but I opted to include it since it is accessible as a route in the API. This makes it easier to work with if that path's output structure ever changes in the future.

Reply:
To me, the way we build this class implies that the response JSON from that endpoint looks like:
```json
{
  "AccessRights": ["Read", "Write"]
}
```

When in reality, the response is just:
```json
["Read", "Write"]
```

The `toJson` and `fromJson` methods appropriately just read it in and map it to an array, but the class itself has this `AccessRights` field that doesn't actually exist anywhere.

I would prefer to just return the list without mapping to a class. @kelseyduffy or @derekendres, any opinions?

Reply:
agree

Reply:
I removed the AccessRights class and just used a ```list[CommonAccessRightsEnum]```